From private repo

v-alje opened the pull request.

PRmerger commented:
@AlekseiPolkovnikov : Thanks for your contribution to the Azure documentation! The author, @tfitzmac, has been notified to review your proposed change.

tfitzmac commented:
@AlekseiPolkovnikov - thanks for making these changes. I like the suggestions, and I agree it clarifies how the function works . I submitted a PR to your branch with a few changes. I fixed one typo. I changed "down to" to "scoped to" - I think either one works so if you want to keep "down to", that is fine. Finally, I removed "limitedly" because I was concerned it might be confusing.

tfitzmac commented:
Looks great.

tfitzmac commented:
#sign-off

Ja-Dunn committed:
edit pass: Add-example-utterances

Commit:
Fixes

Commit:
Fixes.

Commit:
Custom docker daemon docs

Ja-Dunn committed:
edit pass: Add-example-utterances

Ja-Dunn committed:
edit pass: Add-example-utterances

Ja-Dunn committed:
edit pass: Add-example-utterances

Ja-Dunn committed:
edit pass: Add-example-utterances

AshokKumar-P committed:
[ForumsTeam] Minor Typo Corrections

Minor typo corrections

DhruvMsft committed:
Note to make enabling service endpoints explicit

Added a note to ensure that users turn on Service Endpoints. We have been getting a lot of CRIs when users fail to create ACL because they have not turned on Service Endpoints.

DhruvMsft committed:
Note to make enabling service endpoints explicit

Added a note to ensure that users turn on Service Endpoints. We have been getting a lot of CRIs when users fail to create ACL because they have not turned on Service Endpoints.

rolyon committed:
Removing powershell screenshots

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rolepowershell

Ja-Dunn committed:
fix

Ja-Dunn committed:
fixes

jlian committed:
Fix outdated link

Ja-Dunn committed:
fix

Ja-Dunn committed:
fix

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

rolyon committed:
Removed remaining screenshots

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rolepowershell

anzaman committed:
Added samples

rolyon committed:
Updated PowerShell version

anzaman committed:
Update samples-howto-point-to-site-cert-rm-ps.md

anzaman committed:
Update samples-howto-point-to-site-radius-rm-ps.md

anzaman committed:
Update samples-howto-point-to-site-cert-rm-ps.md

Ja-Dunn committed:
fixes

anzaman committed:
Update samples-howto-point-to-site-radius-rm-ps.md

anzaman committed:
Update samples-howto-site-to-site-rm-ps.md

anzaman committed:
Update vpn-gateway-powershell-sample-create-gateway.md

anzaman committed:
Update samples-howto-point-to-site-cert-rm-ps.md

anzaman committed:
Update samples-howto-point-to-site-radius-rm-ps.md

anzaman committed:
Update samples-howto-site-to-site-rm-ps.md

anzaman committed:
Update vpn-gateway-powershell-sample-create-gateway.md

rolyon committed:
Moved List custom roles section, fix to see also links

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rolepowershell

rolyon committed:
Changed examples

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rolepowershell

rolyon committed:
Removed DataActions NotDataActions, tweak to ExpandPrincipalGroups

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rolepowershell

MicrosoftGuyJFlo committed:
[Azure AD] Authentication MVC File Moves - Part 1

MicrosoftGuyJFlo committed:
art

MicrosoftGuyJFlo committed:
link

MicrosoftGuyJFlo committed:
link

MicrosoftGuyJFlo committed:
[Azure AD] Authentication MVC File Moves - Part 2

MicrosoftGuyJFlo committed:
updates

MicrosoftGuyJFlo committed:
updates

MicrosoftGuyJFlo committed:
fix

barclayn committed:
adding blog post entry link

Sogup committed:
Update backup-azure-backup-faq.md

rimman committed:
edits, correction, rewordings in distribute data globally article

Commit:
commit

Commit:
Merge branch 'master' of https://github.com/sujayt/azure-docs-pr

Commit:
update

Commit:
new

carolinacmoravia committed:
L16655: Unrec character in line 79

Hello, @UnifyCloud,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  There is an unrec character that should be deleted. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Review fixes

Commit:
reverted toc structure

Commit:
Review fixes

martincoetzer committed:
Straighting arrows

mrbullwinkle committed:
initial spelling/grammar fix

Commit:
Review fixes

Commit:
Updates

Commit:
Edits.

Commit:
Changed author.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
minor update

rwike77 committed:
updated author

Commit:
Changed pictures.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ddos

Commit:
Changed author.

Commit:
Merge pull request #38241 from Sogup/patch-57

Update backup-azure-backup-faq.md

Commit:
Merge pull request #38125 from jlian/patch-8

Fix outdated link

Commit:
Merge pull request #37789 from AshokKumar-P/patch-26

[ForumsTeam] Minor Typo Correction

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_04_18_2018_continuous_export

Commit:
Merge branch 'master' into master

Commit:
push

mrbullwinkle committed:
add Dale's note'

apwestgarth committed:
Added information to handle custom vnet

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into autoscale

apwestgarth committed:
Added note on adding outbound security rule

When deployed in an existing vnet and using an internal ip for the fileserver we need to have an additional nsg rule added to enable SMB communication from the workernsg to the fileserver.

apwestgarth committed:
Further update to edit for adding outbound security rule to workersnsg

When deployed in an existing vnet and using an internal ip for the fileserver we need to have an additional nsg rule added to enable SMB communication from the workernsg to the fileserver.

anzaman committed:
Rename samples-howto-point-to-site-cert-rm-ps.md to vpn-gateway-sample-point-to-site-certificate-authentication-powershell

anzaman committed:
Rename samples-howto-site-to-site-rm-ps.md to vpn-gateway-sample-site-to-site-powershell.md

anzaman committed:
Rename samples-howto-point-to-site-radius-rm-ps.md to vpn-gateway-sample-point-to-site-radius-authentication-powershell.md

MicrosoftGuyJFlo committed:
[Security] Fixing best practices article acrolinx fail

mrbullwinkle committed:
remove en-us for localization

anzaman committed:
Updating TOC.md

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

PRMerger17 committed:
Merge pull request #38284 from rwike77/autoscale

updated author

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

PRMerger18 committed:
Merge pull request #38289 from apwestgarth/patch-6

Patch 6 - Notes added to deployment document for WorkerNSG

eross-msft committed:
new include file

eross-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lizross-gdpr-stponlyinclude

Commit:
Merge branch 'master' of https://github.com/mani-ramaswamy/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

bandersmsft committed:
azure policy - updated table for clarity and misc fresh

PRMerger20 committed:
Merge pull request #38287 from mrbullwinkle/mrb_04_18_2018_pricing

add Dale's note

MicrosoftGuyJFlo committed:
Merge branch 'master' into AzureADAuthMoves1

bandersmsft committed:
added policy reevaluation sentence

MicrosoftGuyJFlo committed:
Merge branch 'master' into AzureADAuthMoves2

Commit:
Edits

Commit:
Edits.

anzaman committed:
Merge branch 'master' of https://github.com/anzaman/azure-docs-pr

anzaman committed:
Rename vpn-gateway-sample-point-to-site-certificate-authentication-powershell to vpn-gateway-sample-point-to-site-certificate-authentication-powershell.md

PRMerger13 committed:
Merge pull request #38292 from bandersmsft/master

azure policy - updated table for clarity and misc fresh

GitHubber17 committed:
Merge pull request #38255 from sujayt/master

networking-guidance-update

ktoliver committed:
Merge pull request #38207 from MicrosoftGuyJFlo/AzureADAuthMoves1

[Azure AD] Authentication MVC File Moves - Part 1

GitHubber17 committed:
Merge pull request #37657 from mani-ramaswamy/master

Custom docker daemon docs

Juliako committed:
Update media-services-rest-dynamic-manifest.md

rjagiewich committed:
Merge pull request #38245 from rimman/GlobDist

edits, correction, rewordings in distribute data globally article

Commit:
Task 985: Update screen shots to match examples, edits

PRMerger17 committed:
Merge pull request #38231 from barclayn/blog-link

adding blog post entry link

Commit:
Task 985: Fix date

Juliako committed:
Update media-services-rest-how-to-use.md

anzaman committed:
Merge branch 'master' into master

Juliako committed:
Update media-services-rest-manage-entities.md

v-shils committed:
Merge pull request #38265 from davidmu1/tocfix

reverted toc structure

MicrosoftGuyJFlo committed:
Merge branch 'master' into AzureADAuthMoves2

Juliako committed:
Update media-services-rest-encode-asset.md

Juliako committed:
Update media-services-rest-create-contentkey.md

markgalioto committed:
Clean up Backup VMs, redir VM backup article, fix art

barclayn committed:
minor wording fix

jeahyoun committed:
Update context file to point to TOC and breadcrumb.

Juliako committed:
Update media-services-rest-check-job-progress.md

apwestgarth committed:
Update to cover post install outbound sec rule for workers

Updated to cover outbound security rule to enable workers to talk to file server when deployed in an existing vnet and only have internal ip access to file server

rjagiewich committed:
Merge pull request #38268 from martincoetzer/master

Straighting arrows

Juliako committed:
Update media-services-rest-deliver-streaming-content.md

mrbullwinkle committed:
add reference to PowerShell

Juliako committed:
Update media-services-rest-configure-content-key-auth-policy.md

Commit:
Merge pull request #38296 from Juliako/patch-12

Update media-services-rest-dynamic-manifest.md

GitHubber17 committed:
Merge pull request #38273 from jimdial/patch-155

Updates

anzaman committed:
Changed VNet1GWPIP to VNet1GWIP

gatneil committed:
removed outdated registration step

Commit:
Merge pull request #38302 from Juliako/patch-15

Update media-services-rest-encode-asset.md

Commit:
Merge pull request #38304 from Juliako/patch-16

Update media-services-rest-create-contentkey.md

anzaman committed:
Rename vpn-gateway-powershell-sample-create-gateway.md to vpn-gateway-sample-create-gateway.md

anzaman committed:
Rename vpn-gateway-sample-create-gateway.md to vpn-gateway-sample-create-vpn-gateway-powershell.md

Commit:
Merge pull request #38270 from mrbullwinkle/mrb_04_18_2018_continuous_export

initial spelling/grammar fix

Commit:
Merge pull request #38305 from barclayn/ddos-fix02

minor wording fix

apwestgarth committed:
Fixes to improve Acrolinx score

Fixes to improve Acrolinx Score

ktoliver committed:
Merge pull request #38220 from MicrosoftGuyJFlo/AzureADAuthMoves2

[Azure AD] Authentication MVC File Moves - Part 2

MicrosoftGuyJFlo committed:
Merge branch 'master' into FixSomeonesArticle

Commit:
removing repeated instructions across docset

PRMerger13 committed:
Merge pull request #38306 from Juliako/patch-17

Update media-services-rest-check-job-progress.md

PRMerger13 committed:
Merge pull request #38308 from Juliako/patch-18

Update media-services-rest-deliver-streaming-content.md

PRMerger14 committed:
Merge pull request #38309 from Juliako/patch-19

Update media-services-rest-configure-content-key-auth-policy.md

PRMerger14 committed:
Merge pull request #38300 from Juliako/patch-14

Update media-services-rest-manage-entities.md

PRMerger14 committed:
Merge pull request #38298 from Juliako/patch-13

Update media-services-rest-how-to-use.md

tynevi committed:
Merge pull request #38311 from jeahyoun/jeahyoun/msi-context-update

Update context file to point to TOC and breadcrumb.

Juliako committed:
Update media-services-rest-configure-asset-delivery-policy.md

bandersmsft committed:
azure policy - added para for clarity about scope

apwestgarth committed:
Improvements to style, grammar and spelling

As driven by Acrolinx - removed double spaces after full stop, corrected spelling mistakes and made grammar changes

markgalioto committed:
removed backup-azure-arm-vms.md and fixed broken in other file

Commit:
fixed acrolinx feedback

Commit:
Task 985: Edits

PRMerger20 committed:
Merge pull request #38322 from bandersmsft/master

azure policy - added para for clarity about scope

PRMerger20 committed:
Merge pull request #38321 from Juliako/patch-12

Update media-services-rest-configure-asset-delivery-policy.md

Commit:
fixed acrolinx feedback

Commit:
Merge pull request #38262 from carolinacmoravia/patch-682

L16655: Unrec character in line 79

Commit:
Task 985: Edits

Commit:
Merge pull request #38307 from apwestgarth/patch-7

Update to cover post install outbound sec rule for workers

Ja-Dunn committed:
Merge pull request #38282 from jimdial/ddos

DDoS

rjagiewich committed:
Merge pull request #37770 from Ja-Dunn/Add-example-utterances

edit pass: Add-example-utterances

Commit:
Task 985: Fix screenshot outline

Commit:
added PowerShell and Azure CLI script to resources

jlian committed:
Fix incorrect parameter names

tdykstra committed:
add link to async documentation

tdykstra committed:
add link to async documentation

Ja-Dunn committed:
Merge pull request #38286 from jimdial/patch-156

Changed author.

bandersmsft committed:
cost mgt - updated review usage tutorial for usability feedback

GitHubber17 committed:
Merge pull request #38162 from anzaman/master

Added samples

tdykstra committed:
reassign

rjagiewich committed:
Merge pull request #38290 from MicrosoftGuyJFlo/FixSomeonesArticle

[Security] Fixing best practices article acrolinx fail

tdykstra committed:
reassign

bandersmsft committed:
fixed ui text mention

Commit:
Update api-management-howto-use-azure-monitor.md

Ja-Dunn committed:
Merge pull request #38310 from markgalioto/apr182018

Clean up Backup VMs, redir VM backup article, fix art

PRMerger17 committed:
Merge pull request #38327 from SyntaxC4-MSFT/cfowler-local-git

Adding Samples to Additional Resources section

femsulu committed:
(Forums Team) update github user alias

Changed "robb" to "rboucher"

TrishaMc committed:
edit pass: monitoring-usage-and-estimated-costs

PRMerger19 committed:
Merge pull request #38326 from jlian/patch-9

Fix incorrect parameter names

Commit:
Merge pull request #38330 from bandersmsft/master

cost mgt - updated review usage tutorial for usability feedback

ktoliver committed:
Merge pull request #37859 from rolyon/rolyon-rolepowershell

[Azure RBAC] Update manage role assignments with PowerShell

Commit:
Merge pull request #38328 from tdykstra/patch-34

add link to async documentation

Commit:
Merge pull request #38332 from femsulu/patch-12

(Forums Team) update github user alias #7365

mimig1 committed:
Cosmos DB: Added pk to JSON

femsulu committed:
(Forums Team) fix broken link #7186

There's a bad link in this paragraph.
"The properties blob for the Diagnostic Setting follows the format described in this article "insert correct link"

cherylmc committed:
format

PRMerger13 committed:
Merge pull request #37857 from DhruvMsft/patch-3

Note to make enabling service endpoints explicit

PRMerger13 committed:
Merge pull request #37858 from DhruvMsft/patch-4

Note to make enabling service endpoints explicit

Ja-Dunn committed:
Merge pull request #38319 from SyntaxC4-MSFT/cfowler-api-management

[APIM] Fixes MicrosoftDocs/azure-docs#6731

tynevi committed:
Merge pull request #38291 from eross-msft/lizross-gdpr-stponlyinclude

New include file for GDPR articles that don't require the portal for delete or export

Ja-Dunn committed:
Merge pull request #38299 from dksimpson/dean-985

Update screen shots to match examples and edits

PRMerger19 committed:
Merge pull request #38336 from mimig1/patch-239

Cosmos DB: Added pk to JSON

Ja-Dunn committed:
Merge pull request #38339 from cherylmc/toc2

format

Commit:
Merge pull request #38337 from femsulu/patch-13

(Forums Team) fix broken link #7186

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tdykstra committed:
move language ref to first place in next steps

rmca14 committed:
Merge pull request #38333 from TrishaMc/monitoring-usage-and-estimated-costs3

edit pass: monitoring-usage-and-estimated-costs

PRMerger17 committed:
Merge pull request #38354 from tdykstra/patch-35

move language ref to first place in next steps

PRMerger17 committed:
Merge pull request #38312 from gatneil/master

removed outdated registration step

v-alje committed:
Merge pull request #38355 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
